AutoGraph/WebSocketClient.swift

```diff
@@ -324,6 +324,8 @@ extension WebSocketClient: WebSocketDelegate {
                 _ = self.reconnect()
             case .ping, .pong:  // We just ignore these for now.
                 break
+            case .peerClosed:
+                break
```

Comment:
I think we have to try a reconnect here too. this indicates that the server closed the connection, but really the client should just retry indefinitely (with backoff) when the connection ever fails for any reason